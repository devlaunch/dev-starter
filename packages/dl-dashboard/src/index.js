import React from 'react';
import ReactDOM from 'react-dom';

import DashApp from './app';
import * as serviceWorker from './serviceWorker';
import 'antd/dist/antd.css';

ReactDOM.render(<DashApp />, document.getElementById('root'));

// If you want your app to work offline and load faster, you can change
// unregister() to register() below. Note this comes with some pitfalls.
// Learn more about service workers: http://bit.ly/CRA-PWA
serviceWorker.register();
