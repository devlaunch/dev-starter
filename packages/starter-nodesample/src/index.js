import sum from "./lib/sum";
import dbconnect from "./lib/dbconnect";

async function main() {
  console.log(`Sum Results: ${sum(1, 2)}`);
  try {
    const result = await dbconnect(true);
    console.log(` Promise resolved : ${result} `);
  } catch (err) {
    console.log(`Promise rejected ${err.toString()} `);
  }
}

main();
