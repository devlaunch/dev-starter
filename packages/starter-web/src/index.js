import React from 'react';
import ReactDOM from 'react-dom';
import { Provider } from 'react-redux';
import { Route, Switch } from 'react-router-dom';
import { ConnectedRouter } from 'react-router-redux';

import App from './containers/App';
import { store, history } from './modules/redux/store';

ReactDOM.render(
  <Provider store={store}>
    <ConnectedRouter history={history}>
      <Switch>
        <Route path="/" component={App} />
      </Switch>
    </ConnectedRouter>
  </Provider>,
  document.getElementById('root'),
);
