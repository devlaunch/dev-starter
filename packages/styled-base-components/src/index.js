import { A } from './components/A';
import { Abbr } from './components/Abbr';
import { Address } from './components/Address';
import { Article } from './components/Article';
import { Aside } from './components/Aside';
import { B } from './components/B';
import { Blockquote } from './components/Blockquote';
import { Button } from './components/Button';
import { Caption } from './components/Caption';
import { Code } from './components/Code';
import { Dd } from './components/Dd';
import { Dfn } from './components/Dfn';
import { Dialog } from './components/Dialog';
import { Dl } from './components/Dl';
import { Dt } from './components/Dt';
import { Fieldset } from './components/Fieldset';
import { Figcaption } from './components/Figcaption';
import { Figure } from './components/Figure';
import { Footer } from './components/Footer';
import { H1 } from './components/H1';
import { H2 } from './components/H2';
import { H3 } from './components/H3';
import { H4 } from './components/H4';
import { H5 } from './components/H5';
import { H6 } from './components/H6';
import { Header } from './components/Header';
import { Hgroup } from './components/Hgroup';
import { Hr } from './components/Hr';
import { Img } from './components/Img';
import { Input } from './components/Input';
import { Kbd } from './components/Kbd';
import { Legend } from './components/Legend';
import { Label } from './components/Label';
import { Main } from './components/Main';
import { Nav } from './components/Nav';
import { Ol } from './components/Ol';
import { Optgroup } from './components/Optgroup';
import { Output } from './components/Output';
import { P } from './components/P';
import { Pre } from './components/Pre';
import { Progress } from './components/Progress';
import { Samp } from './components/Samp';
import { Section } from './components/Section';
import { Select } from './components/Select';
import { Small } from './components/Small';
import { Strong } from './components/Strong';
import { Sub } from './components/Sub';
import { Summary } from './components/Summary';
import { Sup } from './components/Sup';
import { Svg } from './components/Svg';
import { Table } from './components/Table';
import { Textarea } from './components/Textarea';
import { Th } from './components/Th';
import { Ul } from './components/Ul';

export {
  A,
  Abbr,
  Address,
  Article,
  Aside,
  B,
  Blockquote,
  Button,
  Caption,
  Code,
  Dd,
  Dfn,
  Dialog,
  Dl,
  Dt,
  Fieldset,
  Figcaption,
  Figure,
  Footer,
  H1,
  H2,
  H3,
  H4,
  H5,
  H6,
  Header,
  Hgroup,
  Hr,
  Img,
  Input,
  Kbd,
  Legend,
  Label,
  Main,
  Nav,
  Ol,
  Optgroup,
  Output,
  P,
  Pre,
  Progress,
  Samp,
  Section,
  Select,
  Small,
  Strong,
  Sub,
  Summary,
  Sup,
  Svg,
  Table,
  Textarea,
  Th,
  Ul,
};
