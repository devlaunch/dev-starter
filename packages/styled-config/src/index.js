import screenSize from './config/screen-size';

export default screenSize;
