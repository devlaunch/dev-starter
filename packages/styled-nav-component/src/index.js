import { Nav } from './components/Nav';
import { NavLink } from './components/NavLink';

export {
  Nav,
  NavLink,
};
