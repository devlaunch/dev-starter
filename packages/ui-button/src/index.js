import { Button } from './components/Button';
import { ButtonGroup } from './components/ButtonGroup';
import { ButtonToolbar } from './components/ButtonToolbar';

export { Button, ButtonGroup, ButtonToolbar };
