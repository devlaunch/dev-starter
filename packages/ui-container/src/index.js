import { Container } from './components/Container';

export { Container };
