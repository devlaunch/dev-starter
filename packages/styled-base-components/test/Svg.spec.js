import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Svg } from '../src';

test('Styles matches Svg', () => {
  const tree = renderer.create(<Svg />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule('overflow', 'hidden', {
    modifier: '&:not(:root)'
  });
});
