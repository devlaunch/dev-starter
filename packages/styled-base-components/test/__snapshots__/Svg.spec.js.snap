// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles matches Svg 1`] = `
.c0:not(:root) {
  overflow: hidden;
}

<svg
  className="c0"
/>
`;
